build.sbt

```diff
@@ -168,7 +168,7 @@ lazy val batcher = setupProject(
 lazy val reindex_worker = setupProject(
   project,
   "reindexer/reindex_worker",
-  localDependencies = Seq(big_messaging_typesafe),
+  localDependencies = Seq(mets_adapter),
```

Comment:
Why does the reindex worker depend on the mets adapter?

Reply:
For the MetsSourceData model.

I can move it to a separate library if you'd like? I was thinking maybe `pipeline_model`?

Reply:
ah yes, I see. I think we need a new library for your other PR too but whether it's done in this PR or the other, I don't mind. `pipeline_model` makes it sound like it should be shared within all pipeline services and instead it's something used only by services that deal with source data. Maybe `source_model`?`source_data_model`?

Reply:
Yes, `source_model` is nicer. It will also need MetsSourceData, so I'll add it in the next PR.